packages/api/bin/serve.js

```diff
@@ -127,74 +140,90 @@ function checkEnvVariablesAreSet(moreRequiredEnvVars) {
   });
 }
 
-async function createDBRecords(stackName, user) {
+
+/**
+ * erases Elasticsearch index
+ * @param {any} esClient - Elasticsearch client
+ * @param {any} esIndex - index to delete
+ */
+async function eraseElasticsearchIndices(esClient, esIndex) {
+  try {
+    await esClient.indices.delete({ index: esIndex });
+  } catch (error) {
+    if (error.message !== 'index_not_found_exception') throw error;
+  }
+}
+
+/**
+ * resets Elasticsearch and returns the client and index.
+ *
+ * @param {string} stackName - The name of local stack. Used to prefix stack resources.
+ * @returns {Object} - Elasticsearch client and index
+ */
+async function initializeLocalElasticsearch(stackName) {
   setLocalEsVariables(stackName);
   const esClient = await Search.es(process.env.ES_HOST);
```

Comment:
I would extract these next two lines into a helper called `getESClientAndIndex` or something and have it be used in your seed functions in `serveUtils`. That way, you don't have to extract these values out to be managed in your Cypress code and it's all handled internally to the local API.

Reply:
this is going to cause circular imports unless I move stuff around.